_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -1728,7 +1728,7 @@ public static function get_site_data() {
 			);
 		}
 		if ( $site_data->get_error_code() === 'site_data_fetch_failed' ) {
-			return new WP_Error( 'site_data_fetch_failed', esc_html__( 'Failed fetching site data. Try again later.', 'jetpack' ), array( 'status' => 400 ) );
+			return new WP_Error( 'site_data_fetch_failed', esc_html__( 'Connection to WordPress.com failed. If the problem persists, try reconnecting', 'jetpack' ), array( 'status' => 400 ) );
```

Comment:
Let's keep the verbiage in the context of the failure.
```suggestion
			return new WP_Error( 'site_data_fetch_failed', esc_html__( 'Failed fetching site data from WordPress.com. If the problem persists, try reconnecting Jetpack.', 'jetpack' ), array( 'status' => 400 ) );
```